content/blog/posts/2021-deaths-in-custody-decline-from-previous-year.md

```diff
@@ -0,0 +1,18 @@
+---
+title: 2021 Deaths in Custody decline from previous year
+date: 2022-04-03
+topics:
+  - law enforcement
+  - data oversight
+authors:
+  - Eva Ruth Moravec
+---
+The good news*: the number of people who died in the custody of Texas law enforcement went down in 2021 after peaking the previous year.
+
+The bad news: 2021's total — 904 deaths, according to reports filed with the state of Texas through Feb. 1, 2022 — was still higher than every other year but 2020.
+
+Deaths in custody of law enforcement, including anything from a deadly shooting by a law enforcement officer or an overdose in the back of a patrol car to a death by suicide or medical cause in a prison cell, must be reported to the state of Texas within 30 days of occurring. Based on those reports, which TJI tracks every month, there were an average of 705 Texas deaths in custody from 2010 to 2020. Then, due undoubtedly in part to the 2019 coronavirus, deaths in custody rose almost 40% higher than the 10-year average before dropping slightly in 2021. 
```

Comment:
I can't figure out how to do the in-line text editing so just leaving a comment. My recommended edit to the end of this sentence is:

deaths in custody rose almost 40% higher **in 2020** than the **prior** 10-year average before dropping slightly in 2021.

Also, do you mean to say the average is from 2010-2019 rather than including 2020 in that number?

Reply:
incorporated